django_sharding_library/manager.py

```diff
@@ -50,6 +50,11 @@ def create(self, **kwargs):
 
 class ShardManager(Manager):
 
+    def __init__(self, *args, **kwargs):
+        return_value = super(ShardManager, self).__init__(*args, **kwargs)
+        self.name = 'shard_manager'
```

Comment:
@4word did you manage to use it with Django 1.10 without doing this? I found when it has the name as None, it wouldn't override the options attribute.